.ci_support/win_64_python3.9.____cpythonqt5.yaml

```diff
@@ -15,7 +17,7 @@ python:
 qt:
 - '5'
 qt6_main:
-- '6.6'
+- '6.7'
```

Comment:
sigh it seems that the migration running was flaky:

https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/5894